### PR TITLE
CE-3289: Remove shortcuts entry point from my tools

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
- Remove shortcuts entry point from my tools
- Select first suggestion

### CODE REVIEW

File: extensions/wikia/UserTools/templates/UserTools_Menu.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	<span class="arrow-icon-ctr"><span class="arrow-icon arrow-icon-single"></span></span>
 	<a href="#"><?= htmlspecialchars( $item['caption'] ); ?></a>
 	<ul id="my-tools-menu" class="tools-menu">
-		<?= F::app()->renderView( 'UserTools', 'Menu', array( 'format' => 'html', 'items' => $item['items'] ) ); ?>
+		<?= F::app()->renderView( 'UserTools', 'Menu', [ 'format' => 'html', 'type' => 'submenu', 'items' => $item['items'] ] ); ?>
```
Comment (adamkarminski) [inline extensions/wikia/UserTools/templates/UserTools_Menu.php]: A little bit of docs saying what are the possible types are (submenu/main) would be nice. :)

File: extensions/wikia/UserTools/templates/UserTools_Toolbar.php [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<?= F::app()->renderView( 'UserTools', 'Menu', array( 'format' => 'html', 'items' => $toolbar )); ?>
+<?= F::app()->renderView( 'UserTools', 'Menu', [ 'format' => 'html', 'type' => 'main', 'items' => $toolbar ] ); ?>
```
Comment (adamkarminski) [inline extensions/wikia/UserTools/templates/UserTools_Toolbar.php]: A little bit of docs saying what are the possible types are (submenu/main) would be nice. :)

**Comment by adamkarminski** [comment]
LGTM!

**Comment by azarek** [comment]
LGTM!

**Pull request merged**